pyproject.toml

```diff
@@ -1,3 +1,7 @@
+[build-system]
```

Comment:
does it make a difference to move this block?

Reply:
The hopes are there... :/
I may need to look into this some more time.
Or, alternatively, skip the `requirements.txt` entirely and write everything in the pyproject directly.